packages/hash/integration/src/tests/model/knowledge/link.model.test.ts

```diff
@@ -1,400 +0,0 @@
-          array: true,
-          ordered: false,
-        },
-      ],
-    });
-
-    await Promise.all([
-      EntityModel.create(graphApi, {
-        ownedById: testUser.entityId,
-        entityTypeModel: testEntityType,
-        properties: {},
-        actorId: testUser.entityId,
-      }).then((entity) => {
-        sourceEntityModel = entity;
-      }),
-      EntityModel.create(graphApi, {
-        ownedById: testUser.entityId,
-        entityTypeModel: testEntityType,
-        properties: {},
-        actorId: testUser.entityId,
-      }).then((entity) => {
-        targetEntityFriend = entity;
-      }),
-      EntityModel.create(graphApi, {
-        ownedById: testUser.entityId,
-        entityTypeModel: testEntityType,
-        properties: {},
-        actorId: testUser.entityId,
-      }).then((entity) => {
-        targetEntityAcquaintance = entity;
-      }),
-    ]);
-  });
-
-  let friendLink: LinkModel;
-  let acquaintanceLink: LinkModel;
-
-  it("can link entities", async () => {
-    friendLink = await LinkModel.create(graphApi, {
-      ownedById: testUser.entityId,
-      sourceEntityModel,
-      linkTypeModel: linkTypeFriend,
-      targetEntityModel: targetEntityFriend,
-      actorId: testUser.entityId,
-    });
-
-    acquaintanceLink = await LinkModel.create(graphApi, {
-      ownedById: testUser.entityId,
-      sourceEntityModel,
-      linkTypeModel: linkTypeAcquaintance,
-      targetEntityModel: targetEntityAcquaintance,
-      actorId: testUser.entityId,
-    });
-  });
-
-  it("can get all entity links", async () => {
-    const allLinks = await sourceEntityModel.getOutgoingLinks(graphApi);
-    expect(allLinks).toHaveLength(2);
-    expect(allLinks).toContainEqual(friendLink);
-    expect(allLinks).toContainEqual(acquaintanceLink);
-  });
-
-  it("can get a single entity link", async () => {
-    const links = await sourceEntityModel.getOutgoingLinks(graphApi, {
-      linkTypeModel: linkTypeFriend,
-    });
-
-    expect(links).toHaveLength(1);
-    const link = links[0];
-
-    expect(link?.sourceEntityModel).toEqual(sourceEntityModel);
-    expect(link?.linkTypeModel).toEqual(linkTypeFriend);
-    expect(link?.targetEntityModel).toEqual(targetEntityFriend);
-  });
-
-  it("can remove a link", async () => {
-    await acquaintanceLink.remove(graphApi, { actorId: testUser.entityId });
-
-    const links = await sourceEntityModel.getOutgoingLinks(graphApi, {
-      linkTypeModel: linkTypeAcquaintance,
-    });
-
-    expect(links).toHaveLength(0);
-  });
-
-  let playlistEntityType: EntityTypeModel;
-
-  let playlistEntity: EntityModel;
-
-  let songEntityType: EntityTypeModel;
-
-  let songEntity1: EntityModel;
-
-  let songEntity2: EntityModel;
-
-  let songEntity3: EntityModel;
-
-  let hasSongLinkType: LinkTypeModel;
-
-  let hasSongLink1: LinkModel;
-
-  let hasSongLink2: LinkModel;
-
-  let hasSongLink3: LinkModel;
-
-  it("can create an ordered link", async () => {
```

Comment:
@hashintel/backend I've not restored the tests related to ordering, as they would be broken and because I'm assuming this functionality will be re-implemented in the Graph API rather than the HASH App.

Reply:
I'm not sure that assumption is properly safe yet. Maybe we should capture in an asana task. Not sure which would be appropriate. There's one to look into fractional indexing but this perhaps isn't best suited to that.

Might be worth making a new asana task to "reintroduce ordering link tests in the workspace integration tests"

Reply:
Okay, I've created [a task](https://app.asana.com/0/1202805690238892/1203399804150702/f) but I'm really not sure about the Hash App should be responsible for testing ordering functionality that we want to implement in the Graph API. But perhaps let's discuss in BE refinement